scripts/yocto-build-and-test.sh

```diff
@@ -153,6 +153,11 @@ prepare_build_config() {
     local mender_binary_delta_version=$(get_mender_binary_delta_version)
     cat >> $BUILDDIR/conf/local.conf <<EOF
 PREFERRED_VERSION${sep}pn-mender-binary-delta = "$mender_binary_delta_version"
+
+# Accept the license
+# TODO - This needs to be versioned somehow
+LICENSE_FLAGS_ACCEPTED${sep}append = " commercial_mender-yocto-layer-license"
+
```

Comment:
This should be removed. It is already present [here](https://github.com/mendersoftware/meta-mender/blob/master/tests/build-conf/qemux86-64-uefi-grub/local.conf#L283).